tower-balance/src/pool/mod.rs

```diff
@@ -267,7 +277,7 @@ impl Builder {
         };
 
         Pool {
-            balance: Balance::from_entropy(d),
+            balance: Balance::from_entropy(Box::pin(d)),
```

Comment:
This is unfortunate but this box should not happen that often :(

Reply:
:sob: